packages/zowe-explorer/src/mvs/mvsNodeActions.ts

```diff
@@ -1,82 +0,0 @@
-/*
- * This program and the accompanying materials are made available under the terms of the *
- * Eclipse Public License v2.0 which accompanies this distribution, and is available at *
- * https://www.eclipse.org/legal/epl-v20.html                                      *
- *                                                                                 *
- * SPDX-License-Identifier: EPL-2.0                                                *
- *                                                                                 *
- * Copyright Contributors to the Zowe Project.                                     *
- *                                                                                 *
- */
-
-import * as vscode from "vscode";
-import { ZoweDatasetNode } from "../dataset/ZoweDatasetNode";
-import * as utils from "../utils/ProfilesUtils";
-import { IZoweTree, IZoweDatasetTreeNode } from "@zowe/zowe-explorer-api";
-import { ZoweExplorerApiRegister } from "../ZoweExplorerApiRegister";
-import * as nls from "vscode-nls";
-import * as contextually from "../shared/context";
-
-// Set up localization
-nls.config({
-    messageFormat: nls.MessageFormat.bundle,
-    bundleFormat: nls.BundleFormat.standalone,
-})();
-const localize: nls.LocalizeFunc = nls.loadMessageBundle();
-
-export async function uploadDialog(node: ZoweDatasetNode, datasetProvider: IZoweTree<IZoweDatasetTreeNode>) {
-    const fileOpenOptions = {
-        canSelectFiles: true,
-        openLabel: "Upload File",
-        canSelectMany: true,
-    };
-
-    const value = await vscode.window.showOpenDialog(fileOpenOptions);
-
-    if (value && value.length) {
-        await Promise.all(
-            value.map(async (item) => {
-                // Convert to vscode.TextDocument
-                const doc = await vscode.workspace.openTextDocument(item);
-                await uploadFile(node, doc);
-            })
-        );
-
-        // refresh Tree View & favorites
-        datasetProvider.refreshElement(node);
-        if (contextually.isFavorite(node) || contextually.isFavoriteContext(node.getParent())) {
-            const nonFavNode = datasetProvider.findNonFavoritedNode(node);
-            if (nonFavNode) {
-                datasetProvider.refreshElement(nonFavNode);
-            }
-        } else {
-            const favNode = datasetProvider.findFavoritedNode(node);
-            if (favNode) {
-                datasetProvider.refreshElement(favNode);
-            }
-        }
-    } else {
-        vscode.window.showInformationMessage(localize("enterPattern.pattern", "No selection made."));
-    }
-}
-
-export function getDatasetLabel(node: ZoweDatasetNode) {
```

Comment:
Was this just a remnant from a previous refactoring? I could not find it used anywhere so it seems we're safe to remove it

Reply:
@crawr I'm not quite sure, but like you, I also couldn't find any references to the functions in `mvsNodeActions.ts` outside of the file itself. They seem to be duplicates of functions we have (and are actively using) elsewhere in the code.